man/rhsm.conf.5

```diff
@@ -242,7 +242,7 @@ daemon
 .PP
 splay
 .RS 4
-1 to enable splay. 0 to disable splay. If enabled, this feature delays the initial auto attach and cert check by an amount between 0 seconds and the interval given for the action being delayed. For example if the
```

Comment:
the whole "initial auto attach" needs to be dropped here, not only the "auto"; it's the concept of "attaching" in general that is gone

Reply:
Dropped.